experimental/generator-dotnet-yeoman/generators/app/templates/functions/Dialogs/echo/generated/echo.lu.qna.dialog

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I would assume that we don't need to include this file since it's something that should get generated via "build"-esque process?